tests/services/quality_controller/sequencing_qc_check_scenarios.py

```diff
@@ -0,0 +1,179 @@
+from cg.constants.priority import Priority
```

Comment:
is this file duplicated in `tests/fixture_plugins/quality_controller_fixtures/sequencing_qc_check_scenario.py`?